### DISCUSSION TITLE
How to send an invitation email using automation?

### DISCUSSION THREAD

**Discussion by oitkenmen** [Budibase/budibase#7992] [Help]
Hello, there is an invite link {{ inviteUrl }} in the invitation email template. How can I use it in the automations? Or how to send templates using automations? Is there any binding to the email template? The URL code in {{ inviteUrl }} is temporary, so I cannot just copy it.

**Comment by seoulaja** [comment]
Hi @berkimbayev 

I think you can see in this Budibase University https://vimeo.com/showcase/budibase-university on section 4.1 Automation

**Reply by oitkenmen** [reply]
Hello, sure, they send a usual email, using "Custom" template. The question is how to use other templates, including "Invitation", so that you can send a temporary invite link.
